scripts/shapella-upgrade/deploy-aragon-implementations.js

```diff
@@ -0,0 +1,38 @@
+const runOrWrapScript = require('../helpers/run-or-wrap-script')
+const { log, logSplitter, logWideSplitter, yl, gr } = require('../helpers/log')
+const { saveDeployTx } = require('../helpers/deploy')
+const { readNetworkState, assertRequiredNetworkState } = require('../helpers/persisted-network-state')
+const { deployWithoutProxy, deployBehindOssifiableProxy, updateProxyImplementation } = require('../helpers/deploy')
+const { ZERO_ADDRESS, bn } = require('@aragon/contract-helpers-test')
+
+const { APP_NAMES } = require('../constants')
+
+const DEPLOYER = process.env.DEPLOYER || ''
+const REQUIRED_NET_STATE = [
+  `app:${APP_NAMES.LIDO}`,
+  `app:${APP_NAMES.ORACLE}`,
+  `app:${APP_NAMES.NODE_OPERATORS_REGISTRY}`,
```

Comment:
+ ELRewardsVault maybe (i.e., the protocol underwent the Merge)

Reply:
okay